File: app/src/main/java/com/example/locationsample_kotlin_android/sample/LocationSampleActivity.java
package com.example.locationsample_kotlin_android.sample;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import com.example.locationsample_kotlin_android.R;

public class LocationSampleActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_location_sample);
        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        if (savedInstanceState == null)
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.flFragmentContainer, SampleLocationFragment.newInstance(), SampleLocationFragment.TAG)
                    .commitNow();
    }

}

File: sample/src/main/java/com/linkdev/location_android/sample/LocationSampleActivity.kt
/**
 * Copyright (c) 2020-present Link Development
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
package com.linkdev.location_android.sample

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import com.linkdev.location_android.R

class LocationSampleActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_location_sample)

        val toolbar = findViewById<Toolbar>(R.id.toolbar)
        setSupportActionBar(toolbar)

        if (savedInstanceState == null)
            supportFragmentManager.beginTransaction()
                    .replace(R.id.flFragmentContainer, SampleLocationFragmentLocation.newInstance(), SampleLocationFragmentLocation.TAG)
                    .commitNow()
    }

    override fun onPause() {
        super.onPause()
        println("life: onPause")
    }

    override fun onStop() {
        println("life: onStop")
        super.onStop()
    }

    override fun onDestroy() {
        println("life: onDestroy")
        super.onDestroy()
    }
}